doc/nrf/known_issues.rst

```diff
@@ -518,21 +484,206 @@ Closing sockets
+  The HCI Receiver and Transmitter Test commands are not available.
+
+  **Workaround:** To perform a radio test, use a DTM application:
+
+  * For nRF52, use the DTM application in the nRF5 SDK.
+  * For nRF53, use :ref:`direct_test_mode`.
+
+.. rst-class:: v1-1-0
+
+:option:`CONFIG_BT_HCI_TX_STACK_SIZE` requires specific value
+  :option:`CONFIG_BT_HCI_TX_STACK_SIZE` must be set to 1536 when selecting :option:`CONFIG_BT_LL_SOFTDEVICE`.
+
+.. rst-class:: v1-1-0
+
+:option:`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE` requires specific value
+  :option:`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE` must be set to 2048 when selecting :option:`CONFIG_BT_LL_SOFTDEVICE` on :ref:`central_uart` and :ref:`central_bas`.
+
+.. rst-class:: v1-1-0
+
+:option:`CONFIG_NFCT_IRQ_PRIORITY` requires specific value
+  :option:`CONFIG_NFCT_IRQ_PRIORITY` must be set to 5 or less when selecting :option:`CONFIG_BT_LL_SOFTDEVICE` on :ref:`peripheral_hids_keyboard`.
+
+.. rst-class:: v1-1-0
+
+Wait time required after a directed high duty cycle advertiser times out
+  When selecting :option:`CONFIG_BT_LL_SOFTDEVICE`, if a directed high duty cycle advertiser times out, the application might have to wait a short time before starting a new connectable advertiser.
+  Otherwise, starting the advertiser will fail.
+
+.. rst-class:: v1-1-0
+
+Assert risk after performing a DLE procedure
+  The :ref:`nrfxlib:softdevice_controller` v0.3.0-3.prealpha might assert when receiving a packet with an CRC error on LE Coded PHY after performing a DLE procedure where RX Octets is changed to a value above 140.
+
+.. rst-class:: v1-1-0
+
+Several issues for nRF5340
+  The following issues can occur when using SoftDevice Controller with nRF5340:
+
+  * Poor performance when performing active scanning.
+  * The controller could assert when receiving extended advertising packets.
+  * The ``T_IFS`` could in certain conditions be off by 5 us.
+  * The radio could stay in the TX state longer than expected.
+    This issue can only occur when sending a packet on either LE 1M or LE 2M PHY after receiving or transmitting a packet on LE Coded PHY.
+    If this occurs while performing a Link Layer Control Procedure, the controller could end up retransmitting an acknowledged packet, resulting in a disconnect.
+
+.. rst-class:: v1-1-0 v1-0-0
+
+Sending control packet twice
+  A control packet could be sent twice even after the packet was acknowledged.
+  This would only occur if the radio was forced off due to an unforeseen condition.
+
+.. rst-class:: v1-1-0 v1-0-0
+
+Issues with restarting advertiser
+  The application is unable to restart a connectable advertiser right after a high-duty-cycle advertiser times out.
+
+  **Workaround:** Wait 500 ms before restarting a connectable advertiser.
+
+.. rst-class:: v1-1-0 v1-0-0
+
+Risk of assert on CRC error packet reception
+  The controller could assert when receiving a packet with a CRC error after performing a data length update on Coded PHY.
```

Comment:
"Issues with restarting advertiser" and " Risk of assert on CRC error packet reception" are already mentioned above:
* Wait time required after a directed high duty cycle advertiser times out
* Assert risk after performing a DLE procedure

Those may be combined to:
```
.. rst-class:: v1-1-0 v1-0-0

Wait time required after a directed high duty cycle advertiser times out
  The application is unable to restart a connectable advertiser right after a high-duty-cycle advertiser times out.

  **Workaround:** Wait 500 ms before restarting a connectable advertiser

.. rst-class:: v1-1-0 v1-0-0

Assert risk after performing a DLE procedure
  The controller could assert when receiving a packet with an CRC error on LE Coded PHY after performing a DLE procedure where RX Octets is changed to a value above 140.

Reply:
Thanks, done.